src/components/MTls/MeshMTLSStatus.tsx

```diff
@@ -17,13 +17,12 @@ import { PollIntervalInMs } from '../../types/Common';
 
 type Props = {
   status: string;
-  refreshInterval: PollIntervalInMs;
+  lastRefreshAt: PollIntervalInMs;
   setMeshTlsStatus: (meshStatus: TLSStatus) => void;
 };
 
 type State = {
-  intervalId?: NodeJS.Timeout;
-  refreshInterval: PollIntervalInMs;
+  lastRefreshAt: PollIntervalInMs;
```

Comment:
I don't think you even need the state anymore.  lastRefreshAt is already in redux state, so no need to have it here as well.  If updated in redux it will automatically update this component.